ynput/ayon-core
# Workfiles tool: Author visible in UI

## Changelog Description
Add author username to workfiles tool.

## Additional info
Workfiles tool shows author of workfile based on workfile info on server or author of published version.

It was discovered that workfile info does not fill `createdBy` and `updatedBy` automatically, which is fixed in AYON server 1.1.3, but for backwards compatibility they're both filled and auto-fixed when updating existing file.

The UI shows create user instead of update user, maybe that is not right? Also the UI is "not enough" wide to show all the information, maybe we should make the tool wider by default? Not sure if my workfile template is "production standard" but it does not fit.

## Screenshot
![image](https://github.com/ynput/ayon-core/assets/43494761/51e8748d-a4a5-4006-81b7-4ae676b64f8c)


## Testing notes:
1. Launch a host of your choice.
2. Save workfile via workfiles tool.
3. Open workfile tool again.
4. The new workfile should have author filled with your username.
5. Published workfiles should show author of the representation.
